UnitTests/UnitTests.vcxproj.filters

```diff
@@ -1,36 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <Filter Include="Source Files">
```

Comment:
Probably, you've accidently include these scope of changes. 
Just make another commit with restoring previous state of this file.

Reply:
Clarification: maybe at some point project filters were broken in UnitTests project. You can make those filters again. Search project in Solution Explorer, Add->New Filter. Name one filter - Header Files and another filter - Source Files. Drag files in this project under corresponding filter. Save and commit.
This is not a huge problem, but fix would help to keep project tidy.

Reply:
Done. As i understood, that filters are something like simple folders?

Reply:
Yeap